core/templates/dev/head/domain/collection/CollectionNodeObjectFactorySpec.js

```diff
@@ -103,8 +103,7 @@ describe('Collection node object factory', function() {
     expect(prerequisiteSkillList.getSkillCount()).toEqual(0);
     expect(acquiredSkillList.getSkillCount()).toEqual(0);
     expect(collectionNode.getExplorationSummaryObject()).toEqual({
-      exists: true,
-      newlyCreated: true
+      exists: true
```

Comment:
See above comment. I removed this object when constructing new collection nodes in this way.